internal/core/src/indexbuilder/index_c.cpp

```diff
@@ -84,29 +84,90 @@ CreateIndexV0(enum CDataType dtype,
     return status;
 }
 
+milvus::storage::StorageConfig
+get_storage_config(const milvus::proto::indexcgo::StorageConfig& config) {
```

Comment:
May be duplicated with `indexpb.StorageConfig`?

Reply:
Yes, mainly because index_coord.proto relies on other protos and does not want to generate too many pc.cc files.
We need to discuss whether we should put these in a common proto.